aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { ThemeContext } from 'styled-components';
+import { simpleTags } from './data';
+import {
+  alphabetize,
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from './CreateTag';
+
+const defaultValueOptions = [];
+const defaultValue = '';
+const defaultTags = [];
+
+simpleTags.forEach(datum => {
+  defaultValueOptions.push(datum);
+});
+
+export const CreateTagSimple = () => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  // value
+  const [valueOptions, setValueOptions] = useState(
+    alphabetize(defaultValueOptions),
+  );
+  const [value, setValue] = useState(defaultValue);
+  const [valueSearch, setValueSearch] = useState('');
+
+  // tags
+  const [currentTags, setCurrentTags] = useState(defaultTags);
+
+  return (
+    <AppContainer>
+      <TagPageHeader
+        description={`Tags are values that can be assigned to 
+      resources.`}
+      />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
+              : '100%'
+          }
+          gap={{ column: 'medium', row: 'small' }}
+        >
+          <FormField
+            name="tag-value-simple"
+            htmlFor="tag-value-simple"
+            label="Value"
+            required={{ indicator: false }}
+          >
+            <Select
+              name="tag-value-simple"
+              id="tag-value-simple"
+              placeholder="Select or create a value"
+              value={value}
+              options={valueOptions}
+              onChange={({ option }) => {
+                if (option.includes(prefix)) {
+                  defaultValueOptions.pop(); // remove Create option
+                  defaultValueOptions.push(valueSearch);
+                  setValueOptions(alphabetize(defaultValueOptions));
+                  setValue(valueSearch);
+                } else {
+                  setValue(option);
+                }
+              }}
+              onSearch={text => {
+                const nextValueOptions = defaultValueOptions;
+                updateCreateOption(text, nextValueOptions);
+                const exp = getRegExp(text);
+                setValueOptions(
+                  alphabetize(nextValueOptions.filter(o => exp.test(o))),
+                );
+                setValueSearch(text);
+              }}
+            />
+          </FormField>
+          <Box
+            justify="start"
+            // use theme style values to align Button with FormField input
+            pad={
```

Comment:
Padding / margin is 2px too much. Not immediately clear why this would be... 

![Screen Shot 2022-01-10 at 2 29 55 PM](https://user-images.githubusercontent.com/1756948/148842740-0bb02d55-7e6d-450c-8e7f-1776edcf85d4.png)

Reply:
Hmm I wonder if it's not recalculating properly once we resize down to small and need to grab that `edgeSize` -- looking into it.